dolphinscheduler-python/pom.xml

```diff
@@ -119,7 +119,7 @@
                                     <goal>exec</goal>
                                 </goals>
                                 <configuration>
-                                    <executable>python3</executable>
+                                    <executable>python</executable>
```

Comment:
Change it because someone may have executable name `python3.x` but without `python` soft link to `python3.x`